tests/e2e/performance/test_small_file_workload.py

```diff
@@ -5,24 +5,316 @@
 import pytest
 import time
 
-from ocs_ci.ocs.ocp import OCP
-from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.ocs.ocp import (OCP, get_clustername, get_ocs_version,
+                            get_build, get_ocp_channel, get_provider)
+from ocs_ci.utility.utils import TimeoutSampler, get_ocp_version, run_cmd
 from ocs_ci.ocs.resources.ocs import OCS
 from ocs_ci.utility import templating
 from ocs_ci.ocs.utils import get_pod_name_by_pattern
 from ocs_ci.ocs.ripsaw import RipSaw
 from ocs_ci.ocs import constants
 from ocs_ci.framework.testlib import E2ETest, performance
 from tests.helpers import get_logs_with_errors
+from elasticsearch import (Elasticsearch, exceptions as ESExp)
+import numpy as np
 
 log = logging.getLogger(__name__)
 
 
+class SmallFileResultsAnalyse(object):
+    """
+    This class is reading all test results from elasticsearch server (which the
+    ripsaw running of the benchmark is generate), aggregate them by :
+        test operation (e.g. create / delete etc.)
+          sample (for test to be valid it need to run with more the one sample)
```

Comment:
Indentation on this and next line is not correct.

Reply:
done